ci/env/kava-internal-testnet/genesis.json

```diff
@@ -22,6 +22,8 @@
   },
   "app_hash": "",
   "app_state": {
+    "06-solomachine": null,
```

Comment:
all of the keys added are keys that were missing in the internal-testnet config from a few launches back, but required in order for the genesis file to pass the validation builds